e2e/LongHaul/service/Events/DeviceEventProperties.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Azure.Devices.LongHaul.Service
+{
+    /// <summary>
+    /// Data included in an IoT hub event notification.
+    /// </summary>
+    public class DeviceEventProperties
+    {
+        /// <summary>
+        /// The name of the IoT hub that generated the event.
+        /// </summary>
+        [JsonPropertyName("hubName")]
+        public string HubName { get; set; }
+
+        /// <summary>
+        /// The Id of the device that generated the event.
+        /// </summary>
+        [JsonPropertyName("deviceId")]
+        public string DeviceId { get; set; }
+
+        /// <summary>
+        /// The Id of the module that generated the event.
+        /// </summary>
+        [JsonPropertyName("moduleId")]
+        public string ModuleId { get; set; }
+
+        /// <summary>
+        /// The type of operation published.
+        /// </summary>
+        [JsonPropertyName("opType")]
```

Comment:
Seems that we are not using short term anywhere else, I would prefer name it this way.

```suggestion
        [JsonPropertyName("operationType")]
```

Reply:
This is the data model. I didn't define it. That is what Event Hub emits.